test/jdk/java/awt/dnd/InterJVMGetDropSuccessTest/InterJVMGetDropSuccessTest.java

```diff
@@ -274,19 +276,20 @@ public void run(String[] args) {
                 Thread.sleep(50);
             }
 
+            boolean success1 = false;
             synchronized (Util.SYNC_LOCK) {
                 robot.mouseRelease(InputEvent.BUTTON1_MASK);
```

Comment:
In all uses in this file you can change this to BUTTON1_DOWN_MASK since BUTTON1_MASK is deprecated
https://docs.oracle.com/en/java/javase/21/docs/api/java.desktop/java/awt/event/InputEvent.html#BUTTON1_MASK